doc/source/networking.rst

```diff
@@ -205,6 +205,10 @@ To configure both IPv4 and IPv6:
 
 .. note:: For VNET jails, a default route has to also be specified.
 
+To enable DHCP:
+
+:samp:`# iocage set vnet=on dhcp=on bpf=yes examplejail`
+
```

Comment:
Actually now just setting one prop should work `dhcp=on`. This prop will automatically set the other two props to true.

Reply:
You know...  I should have tried that... :)

Ok, I'll update and repush.  Thanks! :+1:

Reply:
Neat!  This github browser editor is really cool.

Change updated as requested.  Do you need me to squash the commits into one?

Reply:
Actually setting `dhcp=on` is only during creation, during `set` it won't set the others as they may already have values users care about.

Reply:
ok, so just to clarify:

When creating a new clone, you can do 

```
iocage create ... dhcp=on
```

But if you're updating a container that was previously statically set, you need:

```
iocage set vnet=on dhcp=on bpf=yes
```

To ensure all of the appropriate settings are set for DHCP?

Reply:
```suggestion
Creates a FreeBSD 11.0-RELEASE jail with the custom name *myjail* and
sets the jail to start at system boot time.

+ To create a a jail with a DHCP interface add the `dhcp=on` property:
+
+ :samp:`# iocage create -r 11.0-RELEASE --name myjail dhcp=on`
+
+ The `dhcp=on` property implies creating a VNET virtual network stack and
+ enabling the Berkley Packet Filter. DHCP cannot work without VNET.
+ More information about VNET is available in the VNET(9) FreeBSD manual page.

More information about jail properties is available in the iocage(8)
```

(Fixed a couple typos and added a word) Sounds good to me